applications/vessel-history/src/features/search/AdvancedSearch.tsx

```diff
@@ -78,30 +75,34 @@ const AdvancedSearch: React.FC = () => {
   )
 
   return (
-    <div>
+    <div className={styles.container}>
       <div className={styles.row}>
         <InputText
+          inputSize="small"
           onChange={onMainQueryChange}
           value={query}
           label={t('search.shipname', 'Name')}
           autoFocus
           className={styles.half}
         />
         <InputText
+          inputSize="small"
```

Comment:
The `width` of this field is too small, mmsi's have a lenght of 8 and it's currently not being displayed completely. I entered `12345678` for the mmsi but I see only up to the 7.
![image](https://user-images.githubusercontent.com/2205831/133085038-dad0b938-5c42-47a3-a3ef-70341218ffe2.png)